### PR TITLE
New version: AGFEO Dashboard 2.2.0

### PR DESCRIPTION
**Pull request by agfeo-rw**
Checklist for Pull Requests
- [x ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/192343)

### CODE REVIEW

**Comment by agfeo-rw** [comment]
@microsoft-github-policy-service agree [company="{AGFEO GmbH & Co. KG}"]

**Comment by agfeo-rw** [comment]
@microsoft-github-policy-service agree

File: manifests/a/AGFEO/AGFEODashboard/2.2.0/AGFEO.AGFEODashboard.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AGFEO.AGFEODashboard
+PackageVersion: 2.2.0
+InstallerLocale: de-DE
```
Comment (mdanish-kh) [inline manifests/a/AGFEO/AGFEODashboard/2.2.0/AGFEO.AGFEODashboard.installer.yaml:6]:
```suggestion
```

The installer doesn't seem to be in German locale. In previous manifests this was specified as `en-US`. I've seen issues with upgrades as WinGet treats this field as a requirement while upgrading, so probably best to just remove it

File: manifests/a/AGFEO/AGFEODashboard/2.2.0/AGFEO.AGFEODashboard.locale.de-DE.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# PublisherSupportUrl:
+PrivacyUrl: https://agfeo.de/datenschutz/
+Author: AGFEO GmbH & Co. KG
+PackageName: AGFEO Dashboard
+# PackageUrl:
+License: Proprietary
+# LicenseUrl:
+Copyright: 2024, AGFEO GmbH & Co. KG
+# CopyrightUrl:
+ShortDescription: Dashboard for use with AGFEO Phone Systems
```
Comment (mdanish-kh) [inline manifests/a/AGFEO/AGFEODashboard/2.2.0/AGFEO.AGFEODashboard.locale.de-DE.yaml:18]: This should ideally be translated in German as it's a `de-DE` locale file

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/a/AGFEO/AGFEODashboard/2.2.0/AGFEO.AGFEODashboard.locale.de-DE.yaml (hunk 2 above)
Comment (agfeo-rw) [reply manifests/a/AGFEO/AGFEODashboard/2.2.0/AGFEO.AGFEODashboard.locale.de-DE.yaml]: I changed the language of the `ShortDescription` to German

**Review by mdanish-kh — APPROVED** [review]

**Comment by mdanish-kh** [comment]
@agfeo-rw Can you mark above conversation as "Resolved"? That would be needed for the auto-merge logic to merge this PR

**Pull request merged**